pyanaconda/installclasses/fedora-server.py

```diff
@@ -30,7 +30,7 @@
 
 class FedoraServerInstallClass(FedoraBaseInstallClass):
     name = "Fedora Server"
-    stylesheet = "/usr/share/anaconda/fedora-server.css"
+    stylesheet = "/usr/share/anaconda/pixmaps/server/fedora-server.css"
```

Comment:
It seems kinda weird to me to place a css file into a folder called pixmaps, which I would expect to contain bitmap images, possibly with some SVG files as well (as those result in pixmaps when rendered).

Reply:
This is where it is stored in fedora-logos at the moment (because there is no more appropriate location in that package, currently). If that changes, we can update this path as well.

Reply:
Fair enough.